metadata/elasticsearch/src/main/java/com/spotify/heroic/metadata/elasticsearch/MetadataBackendKV.java

```diff
@@ -480,6 +482,12 @@ public FilterBuilder visitStartsWith(final StartsWithFilter startsWith) {
             }
 
             @Override
+            public FilterBuilder visitRegex(final RegexFilter regex) {
+                return regexpFilter(TAGS,
+                    regex.getTag() + TAG_DELIMITER + regex.getValue());
```

Comment:
`regex.getTag()` would have to be escaped to avoid interfering with the overall regular expression. Consider a tag name like `.+`.